exercises/macros/tests/macros.rs

```diff
@@ -77,6 +77,16 @@ mod test {
     }
 }
 
+mod renamed {
+    use macros as macros_renamed;
+
+    #[test]
+    #[ignore]
+    fn test_renamed_module() {
+        let _expected = macros_renamed::hashmap!(23 => 623, 34 => 21,);
+    }
```

Comment:
```suggestion
    #[test]
    #[ignore]
    fn test_macro_out_of_scope() {
        let _expected = macros::hashmap!(23 => 623, 34 => 21,);
    }
```

Let's simplify: what this test depends on is the `hashmap` macro being out of scope, which is solved by introducing a new module. Renaming the module just confuses things.

Reply:
I've change the test case, but rather than having two separate modules. I've merged this new test case with the other out of scope one. That required me to move the `use scope::hashmap;` to inside the other testcase.